exe/kubernetes-run

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'kubernetes-deploy'
+require 'kubernetes-deploy/runner_task'
+require 'optparse'
+
+template = "task-runner-template"
+command = nil
+env_vars = []
+
+ARGV.options do |opts|
+  opts.on("--template=TEMPLATE") { |n| template = n }
+  opts.on("--env-vars=ENV_VARS") { |vars| env_vars = n.split(",")}
+  opts.on("--command=COMMAND") { |c| command = [c] }
+  opts.parse!
+end
+
+runner = KubernetesDeploy::RunnerTask.new(
+  namespace: ARGV[0],
+  context: ARGV[1],
+)
+
+ KubernetesDeploy::Runner.with_friendly_errors do
+  runner.run(
+    task_template: template,
+    command: command,
+    args: ARGV[2..-1],
```

Comment:
It actually wouldn't look like that, it would be `kubernetes-run namespace context rake db:migrate` `rake` is actually an argument here. the `COMMAND` is the container entry-point., which is your init process.